test/helpers.js

```diff
@@ -31,6 +33,18 @@ expect.extend({
   }
 });
 
+expect.extend({
+  toHaveActionsMatching(predicate) {
+    expect.assert(
+      this.actual.filter(predicate).length > 0,
+      `Expected dispatched actions to have action matching supplied predicate.${EOL}
+       Actual: ${inspect(this.actual, { depth: 3 })}`
+    );
+
+    return this;
+  }
+});
+
```

Comment:
Added new matcher to compare actions using a comparator function to tidy up some unit tests.